DomClasses/domInteractions.js

```diff
@@ -0,0 +1,67 @@
+import { randomIntFromInterval } from "../Helpers/mathHelper.js"
+
+
+export function scaleDownCircle(circle){
+    let newRadius = (parseInt(circle.circleRadius)-2)+"px"
+    circle.dom.style.width = newRadius
+    circle.dom.style.height = newRadius
+    circle.dom.style.left = (parseInt(circle.dom.style.left)+1)+"px"
+    circle.dom.style.top = (parseInt(circle.dom.style.top)+1)+"px"
+
+    circle.circleRadius = newRadius
+}
```

Comment:
This function does 2 things: calculates a new radius and updates the DOM with the new radius. This file should have one single role, to update the DOM. The generation of the new radius should be the responsibility of the Circle class.

Reply:
Still does 2 things :(